onnxruntime/test/perftest/main.cc

```diff
@@ -43,6 +43,12 @@ int real_main(int argc, char* argv[]) {
   }
   std::random_device rd;
   perftest::PerformanceRunner perf_runner(env, test_config, rd);
+
+  // Exit if user enabled -n option so that they can just the optimized model or EP context model
```

Comment:
does returning 0 still print out session creation metrics?

Reply:
no

---
In reply to: [1486864267](https://github.com/microsoft/onnxruntime/pull/19501#discussion_r1486864267) [](ancestors = 1486864267)